src/payment/strategies/klarnav2/klarnav2-payment-initialize-options.ts

```diff
@@ -0,0 +1,28 @@
+import { KlarnaLoadResponse } from './klarna-payments';
+
+/**
+ * A set of options that are required to initialize the Klarna payment method.
+ *
+ * When Klarna is initialized, a widget will be inserted into the DOM. The
+ * widget has a list of payment options for the customer to choose from.
+ */
+export default interface KlarnaV2PaymentInitializeOptions {
+    /**
+     * The ID of a container which the payment widget should insert into.
+     */
+    container: string;
```

Comment:
Is should be called `containerId`

Reply:
I based on others `initialize-options` files and many of them are using `container` instead of `containerId`, actually there are two that are using `containerId` (`Adyen` and `Stripe`) so I think the standard is `container`